org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/contentmodel/utils/XMLGenerator.java

```diff
@@ -49,6 +51,16 @@ public XMLGenerator(XMLFormattingOptions formattingOptions, String whitespacesIn
 		this.whitespacesIndent = whitespacesIndent;
 		this.lineDelimiter = lineDelimiter;
 		this.canSupportSnippets = canSupportSnippets;
+		this.autoCloseTags = true;

```

Comment:
remove all assignations and instead rewrite this constructor to call
```java
this(formattingOptions, true, whitespacesIndent, lineDelimiter, canSupportSnippets)
```

why is there an unused maxLevel arg?

Reply:
> why is there an unused maxLevel arg?

Not sure why.